server/models/visits.js

```diff
@@ -81,11 +81,15 @@ var visits = {
                 fuzziness: 'AUTO',
                 operator: 'and',
                 fields: [
+                  'title',
                   'extractedTitle',
                   'extractedContent',
+                  'extractedLead',
                   'extractedDescription',
+                  'extractedUrl',
                   'extractedProviderDisplay',
                   'extractedProviderName',
+                  'extractedProviderUrl',
```

Comment:
Looks like you're right: <http://localhost:9200/chronicle/_search?_source=extractedUrl,extractedProviderUrl> (or at least for my local default dataset).